internal/server/presentation/log.go

```diff
@@ -59,6 +71,30 @@ func (h *AMQPLogHandler) HandleLog(msg amqp.Delivery) {
 	h.SendResponse(utils.OK, "OK", msg.ReplyTo, msg.CorrelationId)
 }
 
+// TODO: [Server] Improve the current RPC Implementation to reduct frontend delays
+// https://github.com/okuda-seminar/log_service/issues/85
+func (h *AMQPCTRLogHandler) HandleCTRLog(msg amqp.Delivery) {
+	req, err := ParseAMQPCTRLog(msg)
+	if err != nil {
+		log.Printf("failed to parse CTR log request: %v", err)
+		msg.Nack(false, false)
```

Comment:
Please add comments to clarify why cases are separated between requeueing and not requeueing, ensuring the reasoning is easy to understand.